do_mpc/model.py

```diff
@@ -729,7 +731,7 @@ def v(self, val):
         raise Exception('Cannot set measurement noise directly.')
 
 
-    def set_variable(self, var_type, var_name, shape=(1,1)):
+    def set_variable(self, var_type, var_name, integer=False, shape=(1,1)):
```

Comment:
Also here: Please give a more descriptive name to the function input: e.g. ``Input_type_integer``.